portal/server/utils/multer-filter.utils.js

```diff
@@ -25,4 +25,19 @@ const multerFilter = (req, file, cb) => {
   }
 };
 
-module.exports = { multerFilter, formatBytes };
+const utilisationReportMulterFilter = (req, file, cb) => {
+  const allowedFormatsRegex = new RegExp(`\\.(${FILE_UPLOAD.ALLOWED_FORMATS_UTILISATION_REPORT.join('|')})$`);
```

Comment:
This is what we do elsewhere in DTFS, I think if we want to be better we should do both mimetype checking as well as possibly trying to check the binary of the file in case people are trying to camouflage files as an allowed type